pkg/cmd/pr/shared/commentable.go

```diff
@@ -225,6 +232,50 @@ func updateComment(commentable Commentable, opts *CommentableOptions) error {
 	return nil
 }
 
+func deleteComment(commentable Commentable, opts *CommentableOptions) error {
+	comments := commentable.CurrentUserComments()
+	if len(comments) == 0 {
+		return errNoUserComments
+	}
+
+	lastComment := &comments[len(comments)-1]
+
+	cs := opts.IO.ColorScheme()
+
+	if (opts.IO.CanPrompt() && !opts.Confirmed) {
+		if opts.Interactive {
+			fmt.Fprintf(opts.IO.Out, "%s Deleted comments cannot be recovered.\n", cs.WarningIcon())
+			ok, err := opts.ConfirmDeleteComment(lastComment.Body)
+			if err != nil {
+				return err
+			}
+			if !ok {
+				return errDeleteNotConfirmed
+			}
+		} else {
+			return errors.New("you must use --yes flag to confirm deletion in non-interactive mode")
+		}
+	}
+
+	httpClient, err := opts.HttpClient()
+	if err != nil {
+		return err
+	}
+
+	apiClient := api.NewClientFromHTTP(httpClient)
+	params := api.CommentDeleteInput{CommentId: lastComment.Identifier()}
+	deletionErr := api.CommentDelete(apiClient, opts.Host, params)
+	if deletionErr != nil {
+		return deletionErr
+	}
+
+	if !opts.Quiet {
+		fmt.Fprintln(opts.IO.Out, "Deleted the comment successfully.")
```

Comment:
Two things:
- This should be printed on `opts.IO.ErrOut`.
- The message reads better as "Comment deleted"

```suggestion
		fmt.Fprintln(opts.IO.ErrOut, "Comment deleted")
```